From 530e81dbd5f15e08a9c87d9983cdec957764b343 Mon Sep 17 00:00:00 2001
From: Simon Horman <horms+renesas@verge.net.au>
Date: Wed, 3 Jul 2013 13:38:24 +0900
Subject: [PATCH 0953/1083] Revert "drm/rcar-du: Prepare/unprepare clock"

This reverts commit 28acb4813af649495205d4b9c3cda099be15db6b.
(cherry picked from commit abbfe31a993e8f94a57876d0a902c41be6f08970)

Signed-off-by: Ryo Kataoka <ryo.kataoka.wt@renesas.com>
---
 drivers/gpu/drm/rcar-du/rcar_du_crtc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/rcar-du/rcar_du_crtc.c b/drivers/gpu/drm/rcar-du/rcar_du_crtc.c
index aa7da6e..31ca55e 100644
--- a/drivers/gpu/drm/rcar-du/rcar_du_crtc.c
+++ b/drivers/gpu/drm/rcar-du/rcar_du_crtc.c
@@ -233,7 +233,7 @@ static void rcar_du_crtc_start(struct rcar_du_crtc *rcrtc)
 		return;
 
 	/* Enable clocks before accessing the hardware. */
-	clk_prepare_enable(rcdu->clock);
+	clk_enable(rcdu->clock);
 
 	/* Enable extended features */
 	rcar_du_write(rcdu, DEFR, DEFR_CODE | DEFR_DEFE);
@@ -297,7 +297,7 @@ static void rcar_du_crtc_stop(struct rcar_du_crtc *rcrtc)
 
 	rcar_du_start_stop(rcdu, false);
 
-	clk_disable_unprepare(rcdu->clock);
+	clk_disable(rcdu->clock);
 
 	rcrtc->started = false;
 }
-- 
1.8.3.2

